cli/roger_push.py

```diff
@@ -174,9 +179,13 @@ def repo_relative_path(self, appConfig, args, repo, path):
             return "{0}/{1}/{2}/{3}".format(os.environ.get('PWD', ''),
                                             args.directory, repo_name, path)
 
+    def getContainerName(self, container):
+         return str(container.keys()[0]) if type(container) == dict else container
```

Comment:
Are we leaving ourselves open to issues by assuming a string if it isn't a dict?